cg/clients/freshdesk/constants.py

```diff
@@ -19,3 +19,6 @@ class Priority:
     MEDIUM = 2
     HIGH = 3
     URGENT = 4
+
+
+TO_MANY_REQUESTS_CODE = 429
```

Comment:
```suggestion
TOO_MANY_REQUESTS_CODE = 429
```

Reply:
solved in [f441155](https://github.com/Clinical-Genomics/cg/pull/3328/commits/f441155bc691ac4d12fabbdb15924ac0943a463e)

Reply:
Imported instead